front/types/assistant/agent_run.ts

```diff
@@ -135,7 +136,10 @@ export async function getRunAgentData(
   // Fetch the agent configuration as we need the full version of the agent configuration.
   const agentConfiguration = await getAgentConfiguration(auth, {
     agentId: agentMessage.configuration.sId,
-    agentVersion: agentMessage.configuration.version,
+    // We do define agentMessage.configuration.version for global agent, ignoring this value here.
+    agentVersion: !isGlobalAgentId(agentMessage.configuration.sId)
+      ? agentMessage.configuration.version
+      : undefined,
```

Comment:
`nit` negation is harder to read 😛 

```suggestion
    agentVersion: isGlobalAgentId(agentMessage.configuration.sId)
      ? undefined
      : agentMessage.configuration.version,
```